layers/+lang/reason/README.org

```diff
@@ -0,0 +1,70 @@
+#+TITLE: Reason layer
+
+[[https://reasonml.github.io/img/reason.svg]]
+
+* Description
+
+Spacemacs layer for ReasonML, based around [[https://github.com/reasonml-editor/reason-mode][reason-mode]].
+
+** Features:
+  - Syntax highlighting / indentation
+  - Autocomplete (via `merlin`)
+  - Lint / error display (via `merlin`, and `flycheck` if `syntax-checking` layer is enabled)
+  - REPL via `rtop`
+  - Auto-formatting (via `refmt`)
+
+* Install
+
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =reason= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+This layer depends on the `ocaml` layer which should be installed automatically.
+
+** Merlin
+
+You'll need merlin (binaries `ocamlmerlin` and `ocamlmerlin-reason`) installed
+on your system and on your `PATH` for errors and autocompletion.
+
+If you're familiar with `opam` you can `opam install merlin reason` to get
```

Comment:
Commands to run are easier to select/copy if they are wrapped in a source block:
```sh
#+BEGIN_SRC sh
  opam install merlin reason
#+END_SRC
```